WebApp/autoreduce_webapp/reduction_viewer/views.py

```diff
@@ -260,6 +260,7 @@ def run_summary(_, instrument_name=None, run_number=None, run_version=0):
         run = ReductionRun.objects.get(instrument=instrument,
                                        run_number=run_number,
                                        run_version=run_version)
+        is_skipped = run.status.value == "s"
```

Comment:
So `run.status.value` uses a character to indicate the status?

Reply:
Yes, I think we should create an ENUM and some point then we can just do run.status == Statuses.SKIPPED.
Low priority though I think

Reply:
I like that idea 🙂